### ISSUE TITLE
SingleTileImagery spanning IDL

### ISSUE THREAD

**Issue by BigDog6432** [AnalyticalGraphicsInc/cesium#5840]
Found an issue where the SingleTileImageryProvider does not seem to work if the image spans the International Date Line.  The following Sandcastle ImageryLayer snippet demonstrates the issue: 
var viewer = new Cesium.Viewer('cesiumContainer', {
    imageryProvider : new Cesium.ArcGisMapServerImageryProvider({
        url : 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer'
    }),
    baseLayerPicker : false
});

var layers = viewer.imageryLayers;
var blackMarble = layers.addImageryProvider(Cesium.createTileMapServiceImageryProvider({
    url : 'https://cesiumjs.org/blackmarble',
    credit : 'Black Marble imagery courtesy NASA Earth Observatory',
    flipXY : true // Only old gdal2tile.py generated tilesets need this flag.
}));
blackMarble.alpha = 0.5;
blackMarble.brightness = 2.0;

layers.addImageryProvider(new Cesium.SingleTileImageryProvider({
    url : '../images/Cesium_Logo_overlay.png',
    rectangle : Cesium.Rectangle.fromDegrees(175.0, 28.0, -170.0, 29.75)
}));

What is rendered on the Globe is the Cesium icon and the letter "C".

Jerrold Stoy

**Comment by ggetz** [comment]
Thanks for the bug report @BigDog6432, and thanks for writing up an example. We'll try to look at this soon, there are a few other [international date line related issues](https://github.com/AnalyticalGraphicsInc/cesium/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20idl%20) with clipping that are high priority.

**Comment by greenlaw** [comment]
I'm running into the same issue - are there any potential workarounds I could test while a formal solution is being investigated (other than chopping the image at the dateline and overlaying with two separate `SingleTileImageryProvider` instances)?

**Comment by hpinkos** [comment]
Hi @greenlaw, as a workaround you could try using a `Rectangle` instead.  [See this demo](https://cesiumjs.org/Cesium/Apps/Sandcastle/?src=Rectangle.html&label=Geometries)

**Comment by greenlaw** [comment]
@hpinkos Thanks, will try using the Entity/Rectangle/Material approach and report back if I run into any issues.

**Comment by BigDog6432** [comment]
Hanna,  I tried the Rectangle approach you mentioned and I am not seeing the image spanning the dateline.  Here is a link to my cesium example.  https://cesiumjs.org/Cesium/Apps/Sandcastle/#c=rVTRTtswFP0Vqy9NteCkA8QELRorFUNjbMoQT5GQSW4TD8eubKesTPz7buImDS2MTVseWtn3nOt7jk+yYJosONyDJmMi4Z5MwPCyoNf1ntdP6uVEScu4BN0fHMUylgtkaUgjSCyTmQDkuiYUpOWWg6EsTb2fsST4SFYAOST9CFJiNZNGlAnisMOK3vcdsN1A9IpbPYlSOuWSWTBYWA3Ynk1nWhWnkGkA4+0MhyENffK2/t155xb7NBz4634FdtKciXWziRJK02h6Su+5zU/EPGdeSPcHjvMYy8e17uog+ZfKzypOV7tPtLI4RuoTJlMCP6wuU7Rn7QCzJAee5ZZUIxFVWoEX8P+d2nVODcP6fy/8M6vOoun0csOsDq0Wx5Vc0z4zm1OrIpZyNMHbewJv5H90gg9xqOrBgdaYvKkNt2src7CIfcAnQdDAi9JYcgvEgCUzpVukVSTlZi7YcqtLLXBT74eLk8mn59LQKh2/oHQ3dOBZKZMamIGNVqRrJkrwBs0Ntr3ejAkKDPePmsu2pZZtGXdxhldSV71uNV5mnbDUUdLNvkXfSw2d1GzF66V0vRKu4cGTbB38Nls4KqUBL1gGJnBtby5Upm6c9d/nWf+ZaB12v1YTJsQtS+6+ajUHbZfepsk+mTFhoHu8aRH/3CoRzBg+40mNulrOoZOfrRq9mkbRyfnlVpzcjT4oVVwpb+N+K0jP742MXQo4dtT3vJgrbUmphYcOWigw0Hg5wW2Z3IGlialpo6AhjVK+IDwdx72Nz3rccxqwMiuF+MYfIO4djwLEP6EJhamW2ZcF6OrVQUg+PL5wm5TSUYDLbZZVCi3VnY6/AA

I ended up splitting the image into 2 images.  One which contains everything to the east of the International Date Line and the other which contains everything to the west of the International Date Line.  I then added the SingleTileImageryProviders (one associated with each image) and got past this issue.
